input/templates/partials/myaccount/my-account-sidebar.hbs

```diff
@@ -1,29 +1,29 @@
 <!-- start myaccount/my-account-sidebar.hbs -->
 <div class="my-account-sidebar">
-  <div {{#if positionOne}}id="my-account-mobile-content"{{/if}} class="my-account-sidebar-items {{#if positionOne}}active{{/if}}">
+  <div {{#if myPersonalDetailsTab}}id="my-account-mobile-content"{{/if}} class="my-account-sidebar-items {{#if myPersonalDetailsTab}}active{{/if}}">
     <a href="{{@root.meta._links.myPersonalDetails.href}}">{{i18n "my-account:myAccountSidebar.personalDetails"}}</a>
   </div>
-  <div {{#if positionTwo}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionTwo}}active{{/if}}">
+  <div {{#if changeMyPasswordTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if changeMyPasswordTab}}active{{/if}}">
```

Comment:
I've put change my password as the 2nd option, because I think it belongs near my personal details form. That's where I usually see it at least :)